src/node_os.cc

```diff
@@ -380,6 +380,10 @@ static void GetPriority(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(priority);
 }
 
+static void GetAvailableParallelism(const FunctionCallbackInfo<Value>& args) {
+  unsigned int parallelism = uv_available_parallelism();
+  args.GetReturnValue().Set(parallelism);
```

Comment:
Since it returns int, we can add fast api support to this. Can you add a benchmark to validate if adding a fast api would make this faster? Happy to help.

Reply:
Kinda pointless, innit? You're not going to be calling it a million times.

Reply:
Yea this isn't something I'm really interested in doing, but someone else can pick it up in the future if they want.